docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CustomMappingSample.cs

```diff
@@ -10,71 +13,88 @@ public static void Example()
-            var dataEnumerable = mlContext.Data.CreateEnumerable<SampleInfertDataTransformed>(transformedData, reuseRowObject: true);
-            Console.WriteLine("IsUnderThirty\t Age\t RowNum\t Education\t ...");
+            // Printing the output data.
+            var dataEnumerable = mlContext.Data.CreateEnumerable<TransformedData>(transformedData, reuseRowObject: true);
+            Console.WriteLine("Age\t Salary\t IsUnderThirty");
             foreach (var row in dataEnumerable)
-                Console.WriteLine($"{row.IsUnderThirty}\t {row.Age}\t {row.RowNum}\t {row.Education}\t ...");
+                Console.WriteLine($"{row.Age}\t {row.Salary}\t {row.IsUnderThirty}");
             // Expected output:
-            //  IsUnderThirty   Age     RowNum  Education       ...
-            //  True            26      0       0-5yrs   ...
-            //  False           42      1       0-5yrs   ...
-            //  False           39      2       12+yrs   ...
-            //  False           34      3       0-5yrs   ...
-            //  False           35      4       6-11yrs  ...
-
+            // Age      Salary  IsUnderThirty
+            // 26       40000   True
+            // 35       80000   False
+            // 34       10000   False
+            // 28       100000  True
+        }
 
-            // Here instead we use it as part of a pipeline of estimators.
-            var pipeline = mlContext.Transforms.CustomMapping(mapping, null)
-                .Append(mlContext.Transforms.Concatenate(outputColumnName: "Features", inputColumnNames: new[] { "Parity", "Induced" }))
-                // It is useful to add a caching checkpoint before a trainer that does several passes over the data.
-                .AppendCacheCheckpoint(mlContext)
-                // We use binary FastTree to predict the label column that was generated by the custom mapping at the first step of the pipeline.
-                .Append(mlContext.BinaryClassification.Trainers.FastTree(labelColumnName: "IsUnderThirty"));
+        // Defines only the column to be generated by the custom mapping transformation in addition to the columns already present.
+        public class CustomMappingOutput
+        {
+            public bool IsUnderThirty { get; set; }
+        }
 
-            // We can train the pipeline and use it to transform data.
-            transformedData = pipeline.Fit(trainData).Transform(trainData);
+        // Defines the schema of the input data.
+        public class InputData
+        {
+            public float Age { get; set; }
+            public float Salary { get; set; }
         }
 
-        // This defines only the column to be generated by the transformation in addition to the columns already present.
-        public class OutputRow
+        // Defines the schema of the transformed data, which includes the new column IsUnderThirty.
+        public class TransformedData
```

Comment:
May we change all example classes to private?
```suggestion
        private class TransformedData
```